hadoop-hdds/docs/content/feature/PrefixFSO.md

```diff
@@ -23,29 +23,33 @@ summary: Supports atomic rename and delete operation.
   limitations under the License.
 -->
 
-The prefix based FileSystem optimization feature supports atomic rename and
- delete of any directory at any level in the namespace. Also, it will perform
-  rename and delete of any directory in a deterministic/constant time.
+The prefix-based File System Optimization feature supports atomic rename and delete of any directory at any level in the
+namespace in deterministic/constant time.
 
-Note: This feature works only when `ozone.om.enable.filesystem.paths` is
- enabled which means that Hadoop Compatible File System compatibility is
-  favored instead of S3 compatibility. Some irregular S3 key names may be
-   rejected or normalized.
+This feature can be enabled for each specific bucket that requires it by setting the `--layout` flag
+to `FILE_SYSTEM_OPTIMIZED` at the time of bucket creation.
 
-This feature is strongly recommended to be turned ON when Ozone buckets are
- mainly used via Hadoop compatible interfaces, especially with high number of
-  files in deep directory hierarchy.
+```bash
+ozone sh bucket create /<volume-name>/<bucket-name> --layout FILE_SYSTEM_OPTIMIZED
+```
+
+Note: File System Optimization favors Hadoop Compatible File System over S3 compatibility. Some irregular S3 key names
```

Comment:
`favors Hadoop Compatible File System over S3 compatibility`

Can you please change it to 

`favors Hadoop Compatible File System instead of S3 compatibility`.